libs/core/src/lib/input-group/input-group.component.html

```diff
@@ -39,6 +39,7 @@
                 [attr.aria-controls]="_inputId"
                 [disabled]="disabled"
                 (mousedown)="_preventFocus($event)"
+                (keydown.enter)="$event.preventDefault()"
```

Comment:
This prevents the popover list from opening on enter press, I don't think we want that do we?